src/main/java/com/internetitem/logback/elasticsearch/util/Base64.java

```diff
@@ -0,0 +1,103 @@
+package com.internetitem.logback.elasticsearch.util;
+
+import java.io.ByteArrayOutputStream;
+
+/**
+ * @author Sridharan Kuppa
```

Comment:
I'm not sure the `@author` is appropriate here (not to suggest that you couldn't write a Base64 implementation from scratch, but I suspect it may have come from somewhere else ;)

I know Base64 is pretty standard, but it would be good to know the provenance of this specific implementation - just in case (to remove any worry about licensing issues down the line)